### PR TITLE
fix issues in TorpedoStore

### PR DESCRIPTION
**Pull request by krafcsikgergo**
There were 3 issues in the mentioned file. All of them are fixed

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -14,6 +14,8 @@ public class TorpedoStore {
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
+
```
Comment (sebestyn) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:18]: bro, ide miért kell egy üres sor?

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (sebestyn) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:45]: szép javítás

**Review by sebestyn — CHANGES_REQUESTED** [review]
nagyjából jó, üres sorokra figyelj

**Review by sebestyn — APPROVED** [review]

**Comment by krafcsikgergo** [comment]
Feladat kész

**Comment by krafcsikgergo** [comment]
SonarCloud still has a warning

**Comment by krafcsikgergo** [comment]
Random class security issue resolved.

**Comment by krafcsikgergo** [comment]
Not merged, need to reopen.

**Pull request merged**